template_archlinux/04_install_qubes.sh

```diff
@@ -24,16 +24,11 @@ fi
 export PACMAN_CACHE_DIR PACMAN_CUSTOM_REPO_DIR "ALL_PROXY=$REPO_PROXY"
 
 echo "  --> Enabling x86 repos..."
-su -c "echo '[multilib]' >> $INSTALL_DIR/etc/pacman.conf"
-su -c "echo 'SigLevel = PackageRequired' >> $INSTALL_DIR/etc/pacman.conf"
-su -c "echo 'Include = /etc/pacman.d/mirrorlist' >> $INSTALL_DIR/etc/pacman.conf"
+sed -i '/\[multilib]/,+1 s/^#//' "${INSTALL_DIR}/etc/pacman.conf"
```

Comment:
How `[multilib]` is added there? I don't see it in mine default `pacman.conf`.

Reply:
Default `pacman.conf` in bootstrap includes multilib commented out:

```
# The testing repositories are disabled by default. To enable, uncomment the
# repo name header and Include lines. You can add preferred servers immediately
# after the header, and they will be used before the default mirrors.

#[core-testing]
#Include = /etc/pacman.d/mirrorlist

[core]
Include = /etc/pacman.d/mirrorlist

#[extra-testing]
#Include = /etc/pacman.d/mirrorlist

[extra]
Include = /etc/pacman.d/mirrorlist

# If you want to run 32 bit applications on your x86_64 system,
# enable the multilib repositories as required here.

#[multilib-testing]
#Include = /etc/pacman.d/mirrorlist

#[multilib]
#Include = /etc/pacman.d/mirrorlist
```

Also the wiki has an entry for [enabling multilib](https://wiki.archlinux.org/title/official_repositories#Enabling_multilib) which uncomments these 2 lines.